doc/ref/configuration/master.rst

```diff
@@ -499,11 +499,11 @@ that connect to a master via localhost.
 ``transport``
 -------------
 
-Default: ``zeromq``
+Default: ``ZeroMQ``
```

Comment:
So these defaults as a code setting should remain ``zeromq``. It's the proper noun at the beginning of the second sentence in the description that should be ``ZeroMQ``.